src/auto_archiver/modules/tiktok_tikwm_extractor/tiktok_tikwm_extractor.py

```diff
@@ -0,0 +1,73 @@
+import requests
+from loguru import logger
+from datetime import datetime, timezone
+
+from auto_archiver.core import Extractor
+from auto_archiver.core import Metadata, Media
+
+
+class TiktokTikwmExtractor(Extractor):
+    """
+    Extractor for TikTok that uses an unofficial API and can capture content that requires a login, like sensitive content.
+    """
+
+    def download(self, item: Metadata) -> Metadata:
+        url = item.get_url()
+
+        # detect URLs that we definitely cannot handle
+        if 'tiktok.com' not in item.netloc:
```

Comment:
Probably fine, but for sites that yt-dlp also processes, I've been checking their code for valid URLs. E.g. 

https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/tiktok.py#L623

Would mean you could use:

```
_VALID_URL.match(url)
```

Just an FYI, I think that's overkill for this. Worst case is a URL like "https://example.com/something/tiktok.com" would pass